Alignment/OfflineValidation/macros/FitPVResolution.C

```diff
@@ -335,17 +335,17 @@ void FitPVResolution(TString namesandlabels, TString theDate = "", bool isStrict
   }
 
   // max vertices
-  const int max_n_vertices = std::min(40, VTXBINS);  // take the minimum to avoid overflow
+  const int max_n_vertices = std::min(60, VTXBINS);  // take the minimum to avoid overflow
   std::vector<float> myNVtx_bins_;
   for (float i = 0; i <= max_n_vertices; i++) {
```

Comment:
```suggestion
  for (float i = 0; i < max_n_vertices + 0.5f ; i++) {
```
(I wouldn't trust a check of equalities between float's...)